augly/audio/__init__.py

```diff
@@ -77,7 +80,7 @@
     "high_pass_filter",
     "insert_in_background",
     "invert_channels",
-    "low_pass_filter",
+    "loop" "low_pass_filter",
```

Comment:
```suggestion
    "loop",
    "low_pass_filter",
```
forgot comma and newline